Components/AppTabNavigator/StocksTab.js

```diff
@@ -15,6 +15,7 @@ class StocksTab extends Component {
 
   constructor(props) {
     super(props);
+    this.willFocusSubscription = null;
```

Comment:
Hoping this gets rid of the error saying this.willFocusSubscription.remove() doesn't exist, I don't see the error anymore on my end, but please double check me